spec/models/waste_carriers_engine/order_copy_cards_registration_spec.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module WasteCarriersEngine
+  RSpec.describe OrderCopyCardsRegistration, type: :model do
+    subject(:order_copy_cards_registration) { build(:order_copy_cards_registration, :has_required_data) }
```

Comment:
I thought we weren't using factories in our unit tests?

Reply:
I don't think we can apply it to models stuff like validations and the workflow can't really be tested with doubles

Reply:
plus this is a build so litterally is the same as `described_class.new` in this case